src/storage/txn/scheduler.rs

```diff
@@ -910,7 +859,7 @@ impl<E: Engine> Scheduler<E> {
     /// This method will try to acquire all the necessary latches. If all the necessary latches are
     /// acquired, the method initiates a get snapshot operation for furthur processing; otherwise,
     /// the method adds the command to the waiting queue(s). The command will be handled later in
-    /// `lock_and_register_get_snapshot` when its turn comes.
+    /// `lock_and_get_snapshot` when its turn comes.
```

Comment:
Seems that "The command will be handled later in..." is stale, the command is actually handled in on_snapshot_finished